app/assets/stylesheets/fae/modules/_modal.scss

```diff
@@ -56,16 +55,20 @@
   img {
     max-width: 100%;
     max-height: 100%;
+    margin: auto;
+    display: block;
   }
 }
 
 .simplemodal-close {
+  display: block;
+  height: 30px;
```

Comment:
I'm going to merge this change. This is update is 🔥 so we can look into ways to refactor later.